subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf5.h

```diff
@@ -18,6 +18,8 @@
 
 #if defined(CONFIG_SOC_SERIES_NRF51X)
 #include "radio_nrf51.h"
+#elif defined(CONFIG_SOC_SERIES_NWTSIM_NRFXX)
```

Comment:
just have a #else clause instead?

Reply:
This board is still setting CONFIG_SOC_NRF52832, so it needs to go first in this if.

Reply:
ok. LGTM.

Reply:
I just moved it as the first, just in case we would want to have a simulated nrf51 someday.